conda-forge.yml

```diff
@@ -1,5 +1,8 @@
 bot:
   automerge: true
+  abi_migration_branches:
+  - "8.3xx"
```

Comment:
https://github.com/conda-forge/pythia8-feedstock/tree/8.3xx already exists, so I added it here to make sure it gets ABI migrations.